packages/block-directory/src/components/block-ratings/stars.js

```diff
@@ -24,23 +19,23 @@ function Stars( { rating } ) {
 				stars
 			) }
 		>
-			{ times( fullStarCount, ( i ) => (
+			{ Array.from( { length: fullStarCount } ).map( ( _, i ) => (
```

Comment:
Huh. `Array.from( { length: n } )` is a new one for me; I usually do `Array( n ).fill( 0 )` or so, `.fill` being there to avoid the empty array issue. I suppose this is more readable. Very interesting.

This will even silently produce an empty array (instead of throwing an exception) for invalid array lengths, such as fractional numbers, thus matching the behaviour of `_.times()`. Pretty impressive 👍